### ISSUE TITLE
rc_mlx5 and dc_mlx5 failure on ARMv8

### ISSUE THREAD

**Issue by shamisp** [openucx/ucx#1812]
```[  FAILED  ] rc_mlx5/uct_p2p_err_test.local_access_error/0, where GetParam() = rc_mlx5/mlx5_0:1
[  FAILED  ] rc_mlx5/uct_p2p_err_test.local_access_error/1, where GetParam() = rc_mlx5/mlx5_0:1/loopback
[  FAILED  ] rc_mlx5/uct_p2p_err_test.remote_access_error/0, where GetParam() = rc_mlx5/mlx5_0:1
[  FAILED  ] rc_mlx5/uct_p2p_err_test.remote_access_error/1, where GetParam() = rc_mlx5/mlx5_0:1/loopback
[  FAILED  ] dc_mlx5/uct_p2p_err_test.local_access_error/0, where GetParam() = dc_mlx5/mlx5_0:1
[  FAILED  ] dc_mlx5/uct_p2p_err_test.local_access_error/1, where GetParam() = dc_mlx5/mlx5_0:1/loopback
[  FAILED  ] dc_mlx5/uct_p2p_err_test.remote_access_error/0, where GetParam() = dc_mlx5/mlx5_0:1
[  FAILED  ] dc_mlx5/uct_p2p_err_test.remote_access_error/1, where GetParam() = dc_mlx5/mlx5_0:1/loopback
```

```
MLNX_OFED_LINUX-4.1-1.0.2.0 (OFED-4.1-1.0.2):
```

**Comment by shamisp** [comment]
```
[ RUN      ] rc_mlx5/uct_p2p_err_test.local_access_error/0
/test/gtest/uct/test_p2p_err.cc:111: Failure
Expected: (num_matched) > (0ul), actual: 0 vs 0
No error which contains the string '' has occurred during the test
/test/gtest/uct/uct_test.cc:564: Failure
At offset 0: Expected: 0x2 Got: 0x0
[  FAILED  ] rc_mlx5/uct_p2p_err_test.local_access_error/0, where GetParam() = rc_mlx5/mlx5_0:1 (298 ms)
[ RUN      ] rc_mlx5/uct_p2p_err_test.local_access_error/1
/test/gtest/uct/test_p2p_err.cc:111: Failure
Expected: (num_matched) > (0ul), actual: 0 vs 0
No error which contains the string '' has occurred during the test
/test/gtest/uct/uct_test.cc:564: Failure
At offset 0: Expected: 0x2 Got: 0x0
[  FAILED  ] rc_mlx5/uct_p2p_err_test.local_access_error/1, where GetParam() = rc_mlx5/mlx5_0:1/loopback (221 ms)
[ RUN      ] rc_mlx5/uct_p2p_err_test.remote_access_error/0
/root/ucx/contrib/../test/gtest/uct/test_p2p_err.cc:111: Failure
Expected: (num_matched) > (0ul), actual: 0 vs 0
No error which contains the string '' has occurred during the test
/test/gtest/uct/uct_test.cc:564: Failure
At offset 8: Expected: 0x4 Got: 0x0
[  FAILED  ] rc_mlx5/uct_p2p_err_test.remote_access_error/0, where GetParam() = rc_mlx5/mlx5_0:1 (317 ms)
[ RUN      ] rc_mlx5/uct_p2p_err_test.remote_access_error/1
/test/gtest/uct/test_p2p_err.cc:111: Failure
Expected: (num_matched) > (0ul), actual: 0 vs 0
No error which contains the string '' has occurred during the test
/test/gtest/uct/uct_test.cc:564: Failure
At offset 8: Expected: 0x4 Got: 0x0
[  FAILED  ] rc_mlx5/uct_p2p_err_test.remote_access_error/1, where GetParam() = rc_mlx5/mlx5_0:1/loopback (225 ms)
[----------] 4 tests from rc_mlx5/uct_p2p_err_test (1061 ms total)

```

**Comment by shamisp** [comment]
@yosefe - any update on this ?

**Comment by yosefe** [comment]
@shamisp i don't have a system to reproduce this error on. if it easy to reproduce on your system, can you please run it with UCX_LOG_LEVEL=data and attach the log?

**Comment by shamisp** [comment]
@yosefe I can generate the log, but you do have the system. Ask  @danielj-mellanox

**Comment by yosefe** [comment]
fixed by 19b441e3ebab2dd34a56a9dd761807e3458b2a95
